src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -2,10 +2,10 @@ import React, { useEffect, useState } from "react";
 import PasswordModal from "@reearth/components/molecules/Settings/Account/PasswordModal";
 import Section from "@reearth/components/molecules/Settings/Section";
 import EditableItem from "@reearth/components/molecules/Settings/Project/EditableItem";
+import Field from "@reearth/components/molecules/Settings/Field";
 import Icon from "@reearth/components/atoms/Icon";
-import { styled, useTheme } from "@reearth/theme";
+import { styled, colors } from "@reearth/theme";
```

Comment:
Sorry after our discussion about importing colors like this, @issmail-basel pushed his theme branch and I saw he was removing all directly used colors in this way.
So rather than importing like this and then using like `colors.text.strong` you should be able to use colors through the theme like `${({theme}) => theme.XX.XXXX};`
That was my bad, my apologies